website/remote-content/shared.js

```diff
@@ -148,6 +153,9 @@ This content was generated on <time dateTime="${generationTime.toISOString()}">$
     };
 
 module.exports.getUrls = (documents) =>
-    Object.entries(documents).map(
-        ([repo, { branch }]) => `${repo}/${branch}/README.md`,
-    );
+    Object.entries(documents).flatMap(([repo, { branch, subPages }]) => [
+        `${repo}/${branch}/README.md`,
+        ...(Object.keys(subPages ?? {}).map(
+            (remotePath) => `${repo}/${branch}/${remotePath}`,
+        ) ?? []),
```

Comment:
Now we not only generate 1 url per object, we also generate urls for the subpages